controller/filehandlingcontroller.php

```diff
@@ -75,23 +81,28 @@ public function load($dir, $filename) {
 		try {
 			if (!empty($filename)) {
 				$path = $dir . '/' . $filename;
-				$filecontents = $this->view->file_get_contents($path);
-				if ($filecontents !== false) {
+				// default of 4MB
+				$maxSize = $this->config->getSystemValue("max_size_text_editor", 4194304);
```

Comment:
We should look into https://github.com/owncloud/files_texteditor/pull/57#discussion_r40809970 instead from my PoV.